src/py/flwr/cli/new/templates/app/pyproject.sklearn.toml.tpl

```diff
@@ -0,0 +1,20 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "$project_name"
+version = "1.0.0"
+description = ""
+authors = [
+    { name = "The Flower Authors", email = "hello@flower.ai" },
+]
+license = {text = "Apache License (2.0)"}
+dependencies = [
+    "flwr[simulation]>=1.8.0,<2.0",
+    "flwr-datasets[vision]>=0.0.2,<1.0.0",
+    "scikit-learn>=1.1.1",
+]
+
+[tool.hatch.build.targets.wheel]
+packages = ["."]
```

Comment:
I recall your comment in [this PR](https://github.com/adap/flower/pull/3226#discussion_r1571919929) about adding a newline at the end of the file. Does the template creation automatically add a newline?